packages/workspace/src/tasks-runner/utils.ts

```diff
@@ -1,5 +1,4 @@
 import {
-  NxJsonConfiguration,
```

Comment:
having this unused import here has been throwing an error on CI. not entirely sure why this isn't an issue on `master` even though this branch is rebased and up to date at the time of writing.